Automattic/jetpack
# Add Jetpack Forms dashboard layout

## Proposed changes:


This patch sets up the minimum layout for the upcoming forms dashboard so we can start working on different parts in parallel. It doesn't need to be perfect now as we'll likely be revisiting each part multiple times as we walk through the tasks.

![Screenshot 2023-02-06 at 22 20 09](https://user-images.githubusercontent.com/8056203/217092399-f1131262-f36d-4468-89e9-6604e16357f0.png)

### Other information:


## Jetpack product discussion



pcsBup-Pl-p2

## Does this pull request change what data or activity we track or use?



No.

## Testing instructions:






Enable the new dashboard with the following two filters before testing:

```
add_filter( 'jetpack_contact_form_use_package', '__return_true' );
add_filter( 'jetpack_forms_dashboard_enable', '__return_true' );
```

- Go to  `/wp-admin/admin.php?page=jetpack-forms`.
- You should see the new dashboard layout.
